src/System.Console/src/System/ConsolePal.Unix.cs

```diff
@@ -7,6 +7,9 @@
 using System.Text;
 using System.Runtime.InteropServices;
 
+// Let the test classes talk to internals
+[assembly: System.Runtime.CompilerServices.InternalsVisibleTo("System.Console.Tests, PublicKey=002400000480000094000000060200000024000052534131000400000100010007D1FA57C4AED9F0A32E84AA0FAEFD0DE9E8FD6AEC8F87FB03766C834C99921EB23BE79AD9D5DCC1DD9AD236132102900B723CF980957FC4E177108FC607774F29E8320E92EA05ECE4E821C0A5EFE8F1645C4C0C93C1AB99285D622CAA652C1DFAD63D745D6F2DE5F17E5EAF0FC4963D261C8A12436518206DC093344D5AD293")] 
```

Comment:
I'd suggest putting this in a separate file that's then only included in the Unix build.